src/libaktualizr/uptane/directorrepository.cc

```diff
@@ -46,37 +40,33 @@ bool DirectorRepository::usePreviousTargets() const {
   return !targets.targets.empty() && latest_targets.targets.empty();
 }
 
-bool DirectorRepository::verifyTargets(const std::string& targets_raw) {
+void DirectorRepository::verifyTargets(const std::string& targets_raw) {
   try {
     // Verify the signature:
     latest_targets = Targets(RepositoryType::Director(), Role::Targets(), Utils::parseJSON(targets_raw),
                              std::make_shared<MetaWithKeys>(root));
     if (!usePreviousTargets()) {
       targets = latest_targets;
     }
-  } catch (const Uptane::Exception& e) {
+  } catch (const Uptane::Exception& e) {  // !
```

Comment:
Why the comment/concern?

Reply:
This was just an help for me when I was going through the code. I thought I removed it, will do.